src/actions/history.js

```diff
@@ -18,8 +19,27 @@ export const onHistoryMiddleware = ({ dispatch, getState }) => (next) => (action
     }
 
     const pathZoom = getZoom(action.payload.location.pathname);
-    if (pathZoom !== state.zoom) {
-      dispatch(pushTimelineRange(pathZoom?.start, pathZoom?.end, false));
```

Comment:
@0x7B5 this used to reset the timeline zoom points when you pressed back in the browser, but below I don't think ever executes anymore, or at least not in this case.